test/test_facebook.py

```diff
@@ -151,14 +154,22 @@ def test_invalid_format(self):
 
 class TestAuthURL(FacebookTestCase):
     def test_auth_url(self):
+        default_version = facebook.VALID_API_VERSIONS[0]
+        api = facebook.GraphAPI(version=default_version)
```

Comment:
I try to make the test more clear to understand why we didn`t specify API version but get a link with version 2.4.